apps/ui-kit/src/lib/components/molecules/input/input.classes.ts

```diff
@@ -7,7 +7,7 @@ export const INPUT_TEXT_CLASSES = 'text-neutral-10 dark:text-neutral-92';
 export const INPUT_PLACEHOLDER_CLASSES =
     ' enabled:placeholder:text-neutral-40/40  dark:placeholder:text-neutral-60/40 enabled:dark:placeholder:text-neutral-60/40';
 export const BORDER_CLASSES =
-    'px-md py-sm rounded-lg border border-neutral-80 group-[.enabled]:cursor-text group-[.errored]:border-error-30 hover:group-[.enabled]:border-neutral-50  dark:border-neutral-60 dark:hover:border-neutral-60 dark:group-[.errored]:border-error-80 [&:has(input:focus)]:border-primary-30 [&:has(input:focus)]:dark:border-primary-80';
+    'px-md py-sm rounded-lg border border-neutral-80 group-[.enabled]:cursor-text group-[.errored]:border-error-30 hover:group-[.enabled]:border-neutral-50 dark:border-neutral-20 dark:hover:border-neutral-60 dark:group-[.errored]:border-error-80 [&:has(input:focus)]:border-primary-30 [&:has(input:focus)]:dark:border-primary-80';
```

Comment:
the darkmode input states are still a bit off I think
- unfocused input field is brighter than focused, when you select an input field the color is like it fades out
- having the input focused but with the mouse outside of the input makes it get very bright which seems off